app/test/service.spec.ts

```diff
@@ -0,0 +1,28 @@
+import oasis from '@oasislabs/client';
+
+jest.setTimeout(20000);
+
+describe('Test', () => {
+  let multisig, counter;
+
+  beforeAll(async () => {
+    multisig = await oasis.workspace.Multisig.deploy(
```

Comment:
this code can also be part of the test. Tests should be independent between them. In this case, there could be a subsuite of tests that just test the deployment. For example:

```
describe ('Deployments', () => {
it ('should deploy non confidential', async () => {
    let multisig = await oasis.workspace.Multisig.deploy(
['b8b3666d8fea887d97ab54f571b8e5020c5c8b58'], 1,
      { header: {confidential: false} }
    );

    var multisig_address = Buffer.from(multisig._inner.address).toString('hex');
    counter = await oasis.workspace.MultisigCounter.deploy(multisig_address, {
      header: {confidential: false}
    });

    expect(multisig).toBeTruthy();
    expect(counter).toBeTruthy();
});

it ('should deploy confidential', async () => {
    let multisig = await oasis.workspace.Multisig.deploy(
['b8b3666d8fea887d97ab54f571b8e5020c5c8b58'], 1,
      { header: {confidential: true} }
    );

    var multisig_address = Buffer.from(multisig._inner.address).toString('hex');
    counter = await oasis.workspace.MultisigCounter.deploy(multisig_address, {
      header: {confidential: false}
    });

    expect(multisig).toBeTruthy();
    expect(counter).toBeTruthy();
});
});
```
...

Reply:
Added